openshift_images/configuring-samples-operator.adoc

```diff
@@ -8,6 +8,21 @@ toc::[]
 
 The Cluster Samples Operator, which operates in the `openshift` namespace, installs and updates the {op-system-base-full}-based {product-title} image streams and {product-title} templates.
 
+[IMPORTANT]
+.Cluster Samples Operator is being downsized  
+====
+. No RFEs will be accepted for whole of samples operator
```

Comment:
Not sure about this point. Since we are not completely deprecating yet, we can potentially accept RFE at the PM discretion if it affect s2i samples

Reply:
correct